### PR TITLE
Make GraphQL tests not depend on JSON serialization

### PR DESCRIPTION
**Pull request by arv**
Instead compare the logical value the JSON represents

### CODE REVIEW

**Comment by arv** [comment]
@jesseditson This should fix those errors...

**Comment by arv** [comment]
PTAL

**Comment by arv** [comment]
@kalman Can you look at this... The motivation is that Go 1.8 changes how it serializes to JSON.

File: go/ngql/query_test.go [hunk 1]
```diff
@@ -32,7 +33,16 @@ func (suite *QueryGraphQLSuite) SetupTest() {
...
+
+	var expectedValue interface{}
+	err := json.Unmarshal([]byte(expect), &expectedValue)
+	suite.NoError(err, "Expected value is not valid JSON")
+
+	var actualValue interface{}
+	err = json.Unmarshal(buff.Bytes(), &actualValue)
+	suite.NoError(err)
+
+	suite.Equal(expectedValue, actualValue)
```
Comment (kalman) [inline go/ngql/query_test.go]: Use `JSONEq`? https://godoc.org/github.com/stretchr/testify/assert#JSONEq

Comment (arv) [inline go/ngql/query_test.go]: Sweet.

**Comment by arv** [comment]
TBR

**Comment by arv** [comment]
Too trivial now...

**Pull request merged**